src/consensus.rs

```diff
@@ -366,12 +368,12 @@ async fn who_is(
 }
 
 async fn send_message(
+    toc: TableOfContentRef,
     address: Uri,
     message: RaftMessage,
-    config: Arc<ConsensusConfig>,
 ) -> anyhow::Result<()> {
-    let message_timeout = Duration::from_millis(config.message_timeout_ms);
-    let channel = timeout_channel(message_timeout, address)
+    let channel = toc
+        .get_grpc_pooled_channel(&address)
```

Comment:
It is a little bit confusing that ToC has `get_grpc_pooled_channel` which does `get_or_create_pooled_channel` (with `create` being an important addition). Initially I though you mistyped here and wanted to correct :smile: